### DISCUSSION TITLE
求助！

### DISCUSSION THREAD

**Discussion by paul-way52** [share121/inter-knot#364] [其他]
![214f16896e05297910eab5dcf734ee3a_720](https://github.com/user-attachments/assets/8f8bcc66-d2dc-4d0b-a13d-cd8a888dc69d)

有没有在港区兼职指挥官的绳匠能告诉我这种素材在哪才能获得？

**Comment by xiangjun818** [comment]
嗯呢呢（打大世界，拿到代币以后港口商店买）

**Comment by muguangchengying** [comment]
打大世界，刷大世界体力，获得代币，兑换紫色代币，港口商店进行购买，每月刷新
